### PR TITLE
Add some headers in the Setup -> Display options panel

### PR DESCRIPTION
**Pull request by fasterit**
Improve upon 73f08de

### CODE REVIEW

**Comment by BenBE** [comment]
What about handling that causes these text items to be skipped when navigating between items?

**Comment by fasterit** [comment]
> What about handling that causes these text items to be skipped when navigating between items?

I did not find an elegant way to do that. I tried it (as a test) by making these items skipped on <kbd>up</kbd> and <kbd>down</kbd> keys:
~~~
      if (OptionItem_kind(selected++) == OPTION_ITEM_TEXT) {
         this->super.selected++;
      }
~~~
^ Notice the lack of declaring this key handled so it will end up in moving two lines.

But that doesn't feel / look nice, so I stopped exploring that direction.

We can always add such code if somebody has a clever idea how to make the Panel code aware of "no go areas".

File: DisplayOptionsPanel.c [hunk 1]
```diff
@@ -97,9 +99,11 @@ DisplayOptionsPanel* DisplayOptionsPanel_new(Settings* settings, ScreenManager*
    this->scr = scr;
 
    Panel_setHeader(super, "Display options");
-   Panel_add(super, (Object*) CheckItem_newByRef("Tree view (for the current Screen tab)", &(settings->ss->treeView)));
+   Panel_add(super, (Object*) TextItem_new(String_cat("For current screen tab: ", settings->ss->name)));
```
Comment (BenBE) [inline DisplayOptionsPanel.c:102]: This leaks memory …

Comment (fasterit) [reply DisplayOptionsPanel.c]: Thanks. Fixed.

**Pull request merged**